decoders/netflow/packet.go

```diff
@@ -114,6 +114,22 @@ type Field struct {
 
 	// The length (in bytes) of the field.
 	Length uint16
+
+	// IANA private enterprise number (PEN) of the authority defining the
+	// Information Element identifier in this Template Record.
+	EnterpriseNumber uint32
+}
+
+// IsVariableLength reports whether the Field is variable-length Information
+// Element (RFC 7011 sec. 3.2. Field Specifier Format).
+func (f *Field) IsVariableLength() bool {
+	return f.Length == 0xffff
+}
+
+// IsEnterpriseNumber reports whether the Field is an enterprise-specific
+// Information Element (RFC 7011 sec. 3.2. Field Specifier Format).
+func (f *Field) IsEnterpriseNumber() bool {
```

Comment:
`IsEnterpriseNumber` is misleading. I'd say it's `IsEnterpriseSpecific`.

Reply:
I think `ContainsEnterpriseNumber` sounds better, renamed